step-functions-data-science-sdk/automate_model_retraining_workflow/code/query_training_status.py

```diff
@@ -0,0 +1,37 @@
+import boto3
+import logging
+import json
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+sm_client = boto3.client('sagemaker')
+
+#Retrieve transform job name from event and return transform job status.
+def lambda_handler(event, context):
+
+    if ('TrainingJobName' in event):
+        job_name = event['TrainingJobName']
+
+    else:
+        raise KeyError('Input must include TrainingJobName key!')
```

Comment:
[Minor] To give user a bit more visibility into this, does it make sense to include an example in the error message, as well as the existing event string?

Reply:
I changed the error to be more descriptive as well as print the existing event string. I found it difficult to add an example as the value for the SageMaker training job name could be any string.

Reply:
I was more thinking that you could provide the structure:
"Please provide a training job name using this format: {"event":{"TrainingJobName":"<my_training_job_name>"}.